### PR TITLE
Raspberry pi

### PR DESCRIPTION
**Pull request by strausmann**

### CODE REVIEW

File: device-types/Raspberry Pi/RPI4-MODBP-8GB.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    label: ''
+    description: ''
+interfaces:
+  - name: eth0
+    type: 1000base-t
+    mgmt_only: false
+    label: ''
+    description: ''
+  - name: wlan0
+    type: ieee802.11ad
```
Comment (tioan) [inline device-types/Raspberry Pi/RPI4-MODBP-8GB.yaml:45]: wrong wifi type

**Comment by danner26** [comment]
Do you still need this? @strausmann

**Comment by strausmann** [comment]
Merge this PR, thank you

**Comment by danner26** [comment]
If you still want to contribute we would be happy to review it again. The original maintainers are busy but I am here to make sure the PRs get worked on
If you would like to contribute please feel free to reopen this PR, update your branch against our master, and then fix any issues. From there I will be happy to review it

**Pull request closed without merging**